maintainers-circle/hosting.md

```diff
@@ -0,0 +1,87 @@
+Try for one speaker that has info/science/data/research/teacher and another that
+is a CNCF maintainer that has experience with the subject. 
+
+### Logistics and Other Planning 
+1. Schedule a 15 minute touch base with the speaker over slack or zoom to nail
+down the final structure and get a final run of show together. 
+2. send the speaker a placeholder calendar invite directly vs having them copy
+from CNCF calendar. Invite them to CNCF slack if they are not there already.
+3. Provide the speaker with a run of show a few days prior to the event/ask if 
+they want to meet. 
+
+### Calendar and Promotion 
+1. Ask Amye or another CNCF staff member to change sig-contributor strategy's
+Thursday meeting to [Maintainers Circle] - [topic] in the subject and update
+with content. 
+2. Ask Amye to email maintainers@ cncf list with info about the event. Previous
+emails can be used as a template. Make sure to over emphasize that other core
+decision makers (committee members, etc), reviewers and other maintainer-like
+roles are invited and not just authors and/or solely MAINTAINERS.md. 
+3. Promote in every -dev and contributor slack channel for each project that you 
+can find, #maintainers-circle, #toc, #sig-contributor-strategy
+4. Tweet about it if thats your thing but with attn to CNCF project maintainers
+5. Ask others to spread the word
+
+## Set Up: Day of Event
+1. set up a questions thread in #maintainers-circle slack if you think one might
+be needed
+2. 3+ hours before the event - ping in all -dev, -contributor slack channel for
+projects, #maintainers-circle, #toc, #sig-contributor-strategy 
+3. create a hackmd notes file so attendees can have a way to capture links
+
+## During Event 
+0. Make sure to claim host, breakout room capabilities are working, and 
+recording is turned off 
+1. Welcome everyone as folks join the line, mention that this won't be recorded,
+and that we will start at :03 after. 
+2. Kick off at :03-5 depending on the flow of traffic
+2.5 Welcome message: hello! thank you for coming, 
+not recording but taking notes that will be published later, 
+there will be a breakout session where folks can share/learn/whatever activity 
+they are doing that day, 
+give an overview of the agenda, 
+where folks can ask questions, 
+add a disclaimer that we are not licensed professional doctors, lawyers, etc 
+(unless the speaker is!); this is based on shared experiences.  
+Lastly, if anyone has questions and then start the show.
+3. Manage time for the speaker and then start to  get everyone in breakrooms at 
+the right time
+4. If you go into a breakroom for hosting/moderating, you can see if others come
+in the main room for breakroom assigments by looking at the breakout zoom popup.
+You don't need to stay in that room the entire time. 
+5. Continue the above cycle for another round or whatever the agenda is shown.
+6. Get everyone back into the main room after the time is up from the second
```

Comment:
Are we asking that there always be a second breakout room?

Reply:
strive for it but if content doesn't allow for it then no. the reason for the breakout rooms is so folks can interact; having only one means 15 mins of engagement only per session and targeting 30 seems to be valuable to folks.